### PR TITLE
implement import as

### PR DESCRIPTION
**Pull request by davidalk**
Support import aliases to prevent name clashes. This change implements support for issue #821 

## Type of change

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

### CODE REVIEW

File: rosetta-testing/src/test/java/com/regnosys/rosetta/tests/RosettaParsingTest.xtend [hunk 1]
```diff
@@ -33,6 +33,43 @@ class RosettaParsingTest {
...
+			namespace test
+			type B:
+				a A (1..1)
+		'''
+
+		#[model1, model2].parseRosettaWithNoIssues
+	}	
+
+	@Test
+ 	def void testCanUseAlisesWhenImpoting() {
```
Comment (SimonCockx) [inline rosetta-testing/src/test/java/com/regnosys/rosetta/tests/RosettaParsingTest.xtend:55]: Typo

File: rosetta-testing/src/test/java/com/regnosys/rosetta/validation/RosettaValidatorTest.xtend [hunk 2]
```diff
@@ -30,6 +30,91 @@ class RosettaValidatorTest implements RosettaIssueCodes {
...
+ 			type B:
+ 				a someAlias.A (1..1)
+ 				d D (1..1)
+ 		'''
+
+ 		#[model1, model2].parseRosettaWithNoIssues
+ 	}	
+
+ 	@Test
+ 	def void testCanUserImportAlisesWhenWildcardPresent() {
```
Comment (SimonCockx) [inline rosetta-testing/src/test/java/com/regnosys/rosetta/validation/RosettaValidatorTest.xtend:86]: Typo

Comment (SimonCockx) [reply rosetta-testing/src/test/java/com/regnosys/rosetta/validation/RosettaValidatorTest.xtend:86]: User -> use
Alises

**Review by SimonCockx — APPROVED** [review]

**Comment by hugohills-regnosys** [comment]
https://github.com/finos/common-domain-model/issues/3364

**Pull request merged**